### DISCUSSION TITLE
Q: 手机网页使用waline 的社交登录后没有传输 登录的数据 || Q: The mobile webpage did not transmit the login data after using waline’s social login.

### DISCUSSION THREAD

**Discussion by ab8ing** [walinejs/waline#2699] [Q&A] [answered]
### Describe the bug | 问题描述

如视频所示：

<video src="http://qq.qqdd.pw/2024-09-01%2018-31-22.mp4"></video>

http://qq.qqdd.pw/2024-09-01%2018-31-22.mp4

感觉是手机浏览器局限于窗口和浏览器限制，或者好像没有对接到（因为后面返回到管理页面，没有传到本页）

### Website URL | 问题网站

https://wbgx.pw/li/empty.html

### Where your waline deploy? | 服务部署在哪里？

Deta

### Where your comment data store? | 数据存储在哪里？

TiDB

---
### Describe the bug | Problem description

As shown in the video:

<video src="http://qq.qqdd.pw/2024-09-01%2018-31-22.mp4"></video>

http://qq.qqdd.pw/2024-09-01%2018-31-22.mp4

It seems that the mobile browser is limited to window and browser restrictions, or it seems that it is not connected (because it is not transferred to this page when I return to the management page later)

### Website URL | Problem website

https://wbgx.pw/li/empty.html

### Where your waline deploy? | Where is the service deployed?

Deta

### Where your comment data store? | Where is the data stored?

TiDB

**Comment by lizheming** [comment, marked as answer]
目前已知部分手机浏览器跳转后确实会缺少和原网页的 opener 句柄关系，导致无法跳转回之前的页面。未来计划是将登录这部分的逻辑写到客户端里。

---
It is currently known that some mobile browsers will indeed lack the opener handle relationship with the original web page after jumping, resulting in the inability to jump back to the previous page. The future plan is to write the login logic into the client.

**Reply by ab8ing** [reply]
好的，期待一下

---
Okay, looking forward to it

**Comment by Sunwuyuan** [comment]
同此问题，似乎依然存在

---
The same problem seems to still exist

**Reply by lizheming** [reply]
可以提供下你的录屏么，我应该已经处理了这个问题

---
Can you provide your screen recording? I should have solved this problem
